components/typograpy/Title/title.tsx

```diff
@@ -0,0 +1,31 @@
+import {ReactElement} from "react";
+import {theme, Typography} from "antd";
+
+interface TitleProps {
+  children?: ReactElement | string;
+  as?: ReactElement;
+  size?: "large" | "medium" | "small",
+  lineHeight?: "large" | "medium" | "small",
+}
```

Comment:
Remover **_lineHeight_**, no se va a usar. 👇🏻 
```suggestion
interface TitleProps {
  children?: ReactElement | string;
  as?: ReactElement;
  size?: "large" | "medium" | "small"
}
```